### PR TITLE
Add keycloak and openshift route support.

### PR DESCRIPTION
**Pull request by parc-jason**
See [business-partner-agent PR 434](https://github.com/hyperledger-labs/business-partner-agent/pull/434)

Signed-off-by: Jason Sherman <jsherman@parcsystems.ca>

### CODE REVIEW

File: charts/bpa/values.schema.json [hunk 1]
```diff
@@ -116,23 +196,14 @@
                     "properties": {
                         "bootstrap": {
```
Comment (frank-bee) [inline charts/bpa/values.schema.json:197]: bootstrap properties I have had modified to be used on forms

File: charts/bpa/values.schema.json [hunk 2]
```diff
@@ -116,23 +196,14 @@
...
                                 },
                                 "username": {
-                                    "type": "string",
-                                    "form": true
+                                    "type": "string"
                                 }
-                            },
-                            "form": true
-                        },
-                        "name": {
```
Comment (frank-bee) [inline charts/bpa/values.schema.json:132]: same with name

File: charts/bpa/Chart.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@ name: bpa
 description: The Business Partner Agent allows to manage and exchange master data between organizations.
 type: application
 
-version: 0.3.0-alpha04
+version: 0.3.0-alpha05
 appVersion: 0.1.0-alpha10
```
Comment (frank-bee) [inline charts/bpa/Chart.yaml:7]: I assume for the keycloak features a newer appVersion ( BPA image) is required , not?

**Review by frank-bee — CHANGES_REQUESTED** [review]
some things in the schema and probably the appVersion should be fixed. The rest of the comments is just for discussion purpose

File: charts/bpa/values.schema.json [hunk 4]
```diff
@@ -12,16 +12,7 @@
...
-                    "description": "Agent name. It's helpful to set it in the as <DID>:<Human readable label>, e.g. did:sov:idu:XnEKP3WieQhNP7ugjV5r36:mybpa",
-                    "form": true
-                },
-                "agentSeed": {
-                    "type": "string",
-                    "description": "The agent seed. Will be generated if not defined here.",
-                    "minLength": 32,
-                    "maxLength": 32,
-                    "form": true
+                    "type": "string"
```
[the remark this answers is not in the document]
Comment (parc-jason) [reply charts/bpa/values.schema.json]: I've put the form fields back for agentName, agentSeed, bpa.bootstrap and bpa.name

File: charts/bpa/Chart.yaml (hunk 3 above)
Comment (parc-jason) [reply charts/bpa/Chart.yaml:7]: I only updated the chart version, wasn't sure if we were ready to tag the BPA code yet.  Would we make appVersion match the GitHub release tag? ex. bpa-0.2.0-alpha03?

**Pull request merged**